acquisitions/src/main/resources/domain/mod-orders/orders-junit.feature

```diff
@@ -0,0 +1,31 @@
+Feature: mod-orders integration tests
+
+  Background:
+    * url baseUrl
+    * table modules
+      | name                |
+      | 'mod-orders'        |
+      | 'mod-login'         |
+      | 'mod-permissions'   |
+      | 'mod-configuration' |
+    * print '### testTenant', testTenant
```

Comment:
remove this
* print '### testTenant', testTenant
    * print '### testAdmin', testAdmin
    * print '### testUser', testUser

Reply:
Done